sdk/java/pulumi/src/main/java/com/pulumi/experimental/automation/exceptions/AutomationException.java

```diff
@@ -0,0 +1,17 @@
+// Copyright 2025, Pulumi Corporation
+
+package com.pulumi.experimental.automation.exceptions;
+
+public class AutomationException extends Exception {
```

Comment:
Also note: the other SDKs don't have an `AutomationException`. The base exception type in the other SDKs is [`CommandException`](https://github.com/pulumi/pulumi-java/pull/1618/files#diff-5c4630475530817b84cabf8d7bcee9df81b2a5ae695d3a052c6421c0bff3eff4). However, since we actually have to put the exception in the signature (e.g. `throws AutomationException`) for most methods, it felt better to use a more generic "AutomationException" rather than using `throws CommandException`, for two reasons:
1. There could be an error that isn't `CommandException` (that will be wrapped by `AutomationException`)
2. If we move to another communication mechanism under the covers (instead of CLI invocations), having the error be `AutomationException` gives us flexibility.